src/cli/index.js

```diff
@@ -171,6 +181,11 @@ export function main({
   //
   const run = (): Promise<void> => {
     invariant(command, 'missing command');
+
+    if (warnAboutRunDashDash) {
+      reporter.warn(reporter.lang('dashDashDeprecation'));
```

Comment:
Arguably, this should be an `info` message, not a `warning` since with the `--strict` option it will break and I don't think should break just because you are still using the harmless `--` thing. What do you think?